modules/FindHarfBuzz.cmake

```diff
@@ -61,3 +62,5 @@ if(NOT TARGET HarfBuzz::HarfBuzz)
         IMPORTED_LOCATION ${HARFBUZZ_LIBRARY}
         INTERFACE_INCLUDE_DIRECTORIES ${HARFBUZZ_INCLUDE_DIR})
 endif()
+
+set(HARFBUZZ_LIBRARIES "${HARFBUZZ_LIBRARY}")
```

Comment:
Ah, that's my paranoia about spaces in paths kicking in. I find it hard to remember how each command expands arguments and whether they convert spaces to lists etc. so I just surround everything with quotes overzealously. Fixed.